### DISCUSSION TITLE
doris无法load导入parquet格式的lzo压缩

### DISCUSSION THREAD

**Discussion by liangmartin** [apache/doris#18586] [A - General / Q&A] [answered]
我在hive的建表为
CREATE TABLE `test`(
......
)
PARTITIONED BY ( 
  `...` date)
ROW FORMAT SERDE 
  'org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe' 
WITH SERDEPROPERTIES ( 
  'field.delim'='', 
  'serialization.format'='') 
STORED AS INPUTFORMAT 
  'org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat' 
OUTPUTFORMAT 
  'org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat'
LOCATION
  'hdfs://xxxxx'
TBLPROPERTIES (
  'parquet.compression'='LZO', 
  
紧接着我在doris正常建表，
CREATE TABLE `test_table5` (
  `ds` date NULL,
  `id_card` int(11) NULL,
  `tran_time` varchar(50) NULL,
  `name` varchar(50) NULL,
  `cash` int(11) NULL
) ENGINE=OLAP
AGGREGATE KEY(`ds`, `id_card`, `tran_time`, `name`, `cash`)
COMMENT 'OLAP'
PARTITION BY RANGE(`ds`)
(PARTITION p20220919 VALUES [('0000-01-01'), ('2022-09-20')),
DISTRIBUTED BY HASH(`id_card`) BUCKETS 2
PROPERTIES (
"replication_allocation" = "tag.location.default: 1",
"storage_format" = "V2",
}

此时doris无法识别带有parquet格式的lzo压缩文件

**Comment by caoliang-web** [comment]
lzo 压缩方式目前还不支持

**Reply by liangmartin** [reply]
在编译的时候加入了with_lzo=1的参数，这个是只支持针对单纯的.lzo文件 压缩方式吗

**Reply by caoliang-web** [reply]
这种方式代表是doris支持lzo压缩，目前默认是lz4,和Doris读取hive lzo 压缩方式没关系

**Reply by liangmartin** [reply]
我再冒昧问一下，我们不关注于dorisDB本身存储数据的压缩格式，我们想要做的是从parquet加上lzo方式的hive表中加载数据到doris中，但是在此过程中，发现不能支持parquet+lzo的数据导入

**Reply by caoliang-web** [reply, marked as answer]
目前还不支持
